src/gsi/aeroinfo.f90

```diff
@@ -285,11 +284,6 @@ subroutine aeroinfo_read
 !      All mpi tasks open and read aerosol information file.
 !      Task mype_aero writes information to aerosol runtime file
   
-       if (mype==mype_aero) then
-          open(iout_aero)
```

Comment:
Good question.  At this point, nothing is written to this file but some stuff specifying values for the input file aeroinfo (e.g., file name, satellite number, channel, errors, etc, similar to the stuff in the radinfo file).  No information on the fits to the obs, etc. is written to the file.  I was trying to reduce the number of files created by the GSI by removing those files which have no useful information in them.  

I could modify the code so that it only creates this file when there are input values for the aerosol data in the aeroinfo file.  What do you think?  That is probably better and I will do that.

Reply:
Going to put this file back in when there is a aeroinfo file.

Reply:
But still removes file when no aeroinfo file exists.  Write message to main output file when aeroinfo does not exist.